docs/releasenotes/release-1.130.md

```diff
@@ -52,4 +50,4 @@ We have added support for direct reconciliation to more resources, with opt-in b
 
 ## Bug Fixes:
 
-* [SAMPLE_Issue 3007](https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/3007) ComputeBackendService cannot refer clientTLSPolicy due to invalid format (This is a sample, your actual release note should not contain `SAMPLE_`, otherwise it will be deleted)
\ No newline at end of file
+* Ensure proper [handling](https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/3825) of `SecretManagerSecretVersion` deletion when the associated GCP resource is already marked as "DESTROYED".
```

Comment:
ditto, suggest not adding this bug fix (it causes more user confusion BTW, because SecretVersion is a opt-in feature)

Reply:
hmmm, you remind me, it looks like the direct resource has been proved by the customers, maybe it's time to switch it to direct as default. 😄

Reply:
Removed the bug fix entry.

We need more customer like this!